test/TEST-50-DISSECT/test.sh

```diff
@@ -16,6 +16,7 @@ test_require_bin mksquashfs veritysetup sfdisk
 
 test_append_files() {
     instmods squashfs =squashfs
+    instmods erofs
```

Comment:
well, that would mean we'd have to change the partition .conf files for the DDIs that we ship. But I really want them to default to erofs...

Reply:
We support dropins in repart, just add a dropin for the tests that overrides the format to squashfs?

Reply:
I went a different way now. I added a relatively trivial commit that adds an env var `$SYSTEMD_REPART_OVERRIDE_FSTYPE`. It's really useful for testing things quickly. All it does is when Format= is parsed swaps out one fstype for another. 

It's super useful for quickly testing stuff, both in the CI and locally. I think this is a much nicer approach, since one can do this from the cmdline so quickly.